Unable to install erlang 22.0-rc2

## To reproduce:
On an OSX system, run `asdf install erlang 22.0-rc2`

## Expected behaviour:
Successful build

## Actual behaviour:
```
$ asdf install erlang 22.0-rc2
Extracting source code
Building Erlang/OTP 22.0-rc2 (asdf_22.0-rc2), please wait...
APPLICATIONS DISABLED (See: /Users/stavros/.asdf/plugins/erlang/kerl-home/builds/asdf_22.0-rc2/otp_build_22.0-rc2.log)
 * jinterface     : No Java compiler found
 * odbc           : ODBC library - header check failed

DOCUMENTATION INFORMATION (See: /Users/stavros/.asdf/plugins/erlang/kerl-home/builds/asdf_22.0-rc2/otp_build_22.0-rc2.log)
 * documentation  : 
 *                  fop is missing.
 *                  Using fakefop to generate placeholder PDF files.

Build failed.
 ERLC	../ebin/pubkey_crl.beam
 VSN	../ebin/public_key.app
 VSN	../ebin/public_key.appup
make[3]: Nothing to be done for `opt'.
=== Leaving application public_key
=== Entering application ssl
/Users/stavros/.asdf/plugins/erlang/kerl-home/builds/asdf_22.0-rc2/otp_src_22.0-rc2/lib/ssl/src/deps/ssl.d:1: *** missing separator.  Stop.
make[2]: *** [opt] Error 2
make[1]: *** [opt] Error 2
make: *** [libs] Error 2

Please see /Users/stavros/.asdf/plugins/erlang/kerl-home/builds/asdf_22.0-rc2/otp_build_22.0-rc2.log for full details.
```

## Additional info:
Looks like some printouts are added to the ssl.d file:
```
$ head /Users/stavros/.asdf/plugins/erlang/kerl-home/builds/asdf_22.0-rc2/otp_src_22.0-rc2/lib/ssl/src/deps/ssl.d
asdf is self upgrading shims to new asdf exec ...
asdf: now running erlc
$(EBIN)/dtls_connection.$(EMULATOR): dtls_connection.erl dtls_connection.hrl \
  ssl_connection.hrl ssl_internal.hrl \
  /Users/stavros/.asdf/installs/erlang/21.1.2/lib/public_key-1.6.3/include/public_key.hrl \
  /Users/stavros/.asdf/installs/erlang/21.1.2/lib/public_key-1.6.3/include/OTP-PUB-KEY.hrl \
  /Users/stavros/.asdf/installs/erlang/21.1.2/lib/public_key-1.6.3/include/PKCS-FRAME.hrl \
  ssl_record.hrl ssl_handshake.hrl ssl_srp.hrl ssl_cipher.hrl \
  dtls_handshake.hrl tls_handshake.hrl ssl_api.hrl ssl_alert.hrl \
  dtls_record.hrl \
```
(notice the asdf lines)

--- a/README.md
+++ b/README.md
@@ -141,10 +141,10 @@ for jinterface
 `sudo yum install -y java-1.8.0-openjdk-devel`
 
 ODBC support
-`sudo yum install -y libiodbc unixODBC.x86_64 erlang-odbc.x86_64`
+`sudo yum install -y libiodbc unixODBC-devel.x86_64 erlang-odbc.x86_64`
 
 for the documentation to be built
-`sudo yum install -y libxslt`
+`sudo yum install -y libxslt fop`
 
 ## Solus
 

